docs/guides/latest-updates.mdx

```diff
@@ -14,22 +14,24 @@ Keep up with the latest and greatest from Qiskit and IBM Quantum&trade;! Gathere
 
 {/* release summaries written by product owners */}
 
-### Qiskit SDK v1.1.0
+### Qiskit SDK v1.2.0
 
 *To see the release notes for all versions, visit the [Qiskit SDK release notes](/api/qiskit/release-notes).*
 
-Qiskit SDK v1.1.0 has arrived, marking the first minor version release of the Qiskit SDK’s first major version. On the IBM Quantum blog, we review some of the key highlights — including top new features and improvements, as well as some important deprecations. Read the blog [here](https://www.ibm.com/quantum/blog/qiskit-1-1-release-summary).
+Qiskit 1.2 (stable release) is now available on [PyPI](https://pypi.org/project/qiskit/1.2.0/) and [GitHub](https://github.com/Qiskit/qiskit/releases/tag/1.2.0). Thanks to the whole dev team and all external contributors!
```

Comment:
Nits:

1. Not sure `whole` is bringing much value?
2. Probably better to call it `open-source` rather than `external`

```suggestion
Qiskit 1.2 (stable release) is now available on [PyPI](https://pypi.org/project/qiskit/1.2.0/) and [GitHub](https://github.com/Qiskit/qiskit/releases/tag/1.2.0). Thanks to the maintainers and open-source contributors!
```